Tests/Unit/MiniAppScriptMessageHandlerTests.swift

```diff
@@ -1317,6 +1376,45 @@ class MiniAppScriptMessageHandlerTests: QuickSpec {
                     expect(mockCallbackProtocol.response).toEventually(equal("{\"used\":0,\"max\":2000000}"))
                 }
             }
+            context("when MiniAppScriptMessageHandler receives sendJsonToHostApp info message") {
+                let scriptMessageHandler = MiniAppScriptMessageHandler(
+                    delegate: callbackProtocol,
+                    hostAppMessageDelegate: mockMessageInterface,
+                    adsDisplayer: mockAdsDelegate,
+                    secureStorageDelegate: mockSecureStorageDelegate,
+                    miniAppManageDelegate: mockMiniAppManageInterface,
+                    miniAppId: mockMiniAppInfo.id,
+                    miniAppTitle: mockMiniAppTitle
+                )
+                it("will return SUCCESS after host app recieves valid content strig") {
+                    mockMessageInterface.messageContentAllowed = true
+                    let command = "{\"action\":\"sendJsonToHostapp\",\"param\":{\"jsonInfo\":{\"content\":\"{\\\"data\\\":\\\"Thisisasamplejsoninformation\\\"}\"}},\"id\":\"10.050192665128856\"}"
```

Comment:
same here and the other tests `\\\` can be one backslash

Reply:
As per my comments earlier these `\\\` are needed. See below the required final json payload sent to MiniApp via JS-Bridge. `{"action":"sendJsonToHostapp","param":{"jsonInfo":{"content":"{\"data\":\"This is a sample json information\"}"}},"id":"11.172421055204431"}` . With the above `\\\` this will be the end results if we remove them then will result in parsing error.